src/Objs/Redirect/RedirectObjClass.ts

```diff
@@ -0,0 +1,9 @@
+import { provideObjClass } from 'scrivito'
+
+export const Redirect = provideObjClass('Redirect', {
```

Comment:
Please add `Redirct` to `BLACKLIST_OBJ_CLASSES` - so that it will not show up in search results.